src/ConsoleStyle.fs

```diff
@@ -0,0 +1,282 @@
+    let subTitle (subTitle: string): unit =
+        Console.WriteLine(subTitle, color Color.SubTitle)
+
+    [<CompiledName("SubTitlef")>]
+    let subTitlef (format: Printf.StringFormat<('a -> string)>) (value: 'a): unit =
+        value
+        |> sprintf format
+        |> subTitle
+
+    [<CompiledName("Error")>]
+    let error (message: string): unit =
+        Console.WriteLine(message, color Color.Error)
+        newLine()
+
+    [<CompiledName("Errorf")>]
+    let errorf (format: Printf.StringFormat<('a -> string)>) (message: 'a): unit =
+        message
+        |> sprintf format
+        |> error
+
+    [<CompiledName("Success")>]
+    let success (message: string): unit =
+        Console.WriteLine(message, color Color.Success)
+        newLine()
+
+    [<CompiledName("Successf")>]
+    let successf (format: Printf.StringFormat<('a -> string)>) (message: 'a): unit =
+        message
+        |> sprintf format
+        |> success
+
+    [<CompiledName("Indentation")>]
+    let indentation: string =
+        String.replicate 4 " "
+
+    [<CompiledName("Indent")>]
+    let indent (value: string): string =
+        indentation + value
+
+    [<CompiledName("Indentf")>]
+    let indentf (format: Printf.StringFormat<('a -> string)>) (value: 'a): string =
+        value
+        |> sprintf format
+        |> indent
+
+    //
+    // Output many
+    //
+
+    [<CompiledName("Messages")>]
+    let messages (prefix: string) (messages: seq<string>): unit =
+        messages
+        |> Seq.map (sprintf "%s%s" prefix)
+        |> Seq.iter message
+
+    let private optionsList maxLength title options =
+        subTitle title
+        options
+        |> Seq.map (fun (command, description) ->
+            sprintf "- %-*s %-s" (maxLength + 1) command description
+        )
+        |> messages indentation
+
+    [<CompiledName("Options")>]
+    let options (title: string) (options: seq<string * string>): unit =
+        optionsList (options |> getMaxLengthForOptions) title options
+        newLine()
+
+    [<CompiledName("Optionsf")>]
+    let optionsf (format: Printf.StringFormat<('a -> string)>) (title: 'a) (options: seq<string * string>): unit =
+        options
+        |> optionsList (options |> getMaxLengthForOptions) (title |> sprintf format)
+        newLine()
+
+    [<CompiledName("List")>]
+    let list (messages: seq<string>): unit =
+        messages
+        |> Seq.map (sprintf " - %s")
+        |> Seq.iter message
+
+    //
+    // Complex components
+    //
+
+    [<CompiledName("Table")>]
+    let table (header: seq<string>) (rows: seq<seq<string>>): unit =
+        let renderHeaderRow (row: string) =
+            Console.WriteLine(row, color Color.TableHeader)
+
+        Table.renderTable renderHeaderRow header rows
+        newLine()
+
+    [<CompiledName("ProgressStart")>]
+    let progressStart (initialMessage: string) (total: int): ProgressBar =
+        let options = new ProgressBarOptions()
```

Comment:
you can use object initializer here:
        let options = 
            new ProgressBarOptions(
                ForegroundColor = ConsoleColor.Yellow, 
                ForegroundColorDone = Nullable<ConsoleColor>(ConsoleColor.DarkGreen))